Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -6,10 +6,13 @@
 #include <QSettings>
 #include <QSize>
 
+#include <iostream>
```

Comment:
This can be removed. Note that this causes [code](https://godbolt.org/g/q48TYs) to be executed even when it isn't used, because most iostream implementations inject a static constructor into the translation unit its included into.

Reply:
Would have removed it anyway as it is a leftover but thanks for the extra information!